README.adoc

```diff
@@ -0,0 +1,274 @@
+= Atomic Developer Bundle
+:toc:
+
```

Comment:
@Preeticp I already mentioned this when I submitted the PR: the problem with the TOC being live-rendered here is apparently unrelated to the TOC in VSM, because the version of asciidoc changed and GitHub doesn't render TOCs anymore. I already made a comment about this and @bexelbie agreed that we shouldn't invest in it (I did a bunch of research and it looks like there are some really ugly hacks but no out-of-the-box way to do it now). 

@hferentschik I do use the ASCIIDoctor browser plugin as well as the Atom preview extension to test each file. This is how I know that the TOC will render correctly when we publish documentation to the final site. GitHub's TOC problem stems from its "safety" features where it disables some markup tags (other formats have their own problems, it's quite a fun ride). At this time, because we don't have a publication system, we cannot test the entire published doc set, but I do test everything locally before I push to the PR.